.github/workflows/check.yaml

```diff
@@ -13,11 +13,14 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: ${{ env.GO_VERSION }}
-      - uses: golangci/golangci-lint-action@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
```

Comment:
The `gofmt` and `goimport` can be run as part of the  `golangci-lint` [1], let's just use that and active the linters that we want

[1] https://github.com/golangci/golangci-lint/blob/b3f97632464b050f8893314649d0c3f06ad66872/.golangci.yml#L79-L80